api/src/pcapi/notifications/push/__init__.py

```diff
@@ -10,6 +10,7 @@
 class BatchEvent(enum.Enum):
     USER_DEPOSIT_ACTIVATED = "user_deposit_activated"
     USER_IDENTITY_CHECK_STARTED = "user_identity_check_started"
+    HAS_ADDED_OFFER_TO_FAVOURITES = "has_added_offer_to_favourites"
```

Comment:
```suggestion
    HAS_ADDED_OFFER_TO_FAVORITES = "has_added_offer_to_favorites"
```
Je veux bien qu'on ne mette pas le 'u' par cohérence avec la relation qui s'appelle `Favorite`.